docs/tutorial/comments.md

```diff
@@ -2,20 +2,16 @@
 sidebar_position: 2
 ---
 # Comments
-
-1. Comments can be used to explain Unv code.
-2. Comments can be used to prevent execution when testing code.
+---
+Comments are used to explain code or to tell the compiler or runtime to ignore these lines.Thses lines are esentially invisible to the compiler or runtime.
 
 ## Creating a Comment
 
 Comments starts with a #, and Unv will ignore them:
 
 **Example**
 ```py {1}
-import print from 'standard'
-
 #This is a comment
```

Comment:
```suggestion
#This is a comment
print("Hello, World!")
```
Added for clarity. Should we remove it?

Reply:
I don't think so , the hello world shows that that will be executed but nothing else will be.

Reply:
OK